_posts/2019-02-28-converting-legacy-properties-files-into.adoc

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
Not sure if the diff is rendering in a weird way but it looks like this file appears here and below?